packages/types/src/electra/sszTypes.ts

```diff
@@ -61,8 +61,8 @@ export const AttestingIndices = new ListBasicType(
 export const Attestation = new ContainerType(
   {
     aggregationBits: AggregationBits, // Modified in ELECTRA
-    data: phase0Ssz.AttestationData,
     committeeBits: CommitteeBits, // New in ELECTRA
```

Comment:
shouldn't `committeeBits` now be appended after the signature?

Reply:
> shouldn't `committeeBits` now be appended after the signature?

Sorry still working on it